lib/cobra_commander/git_changed.rb

```diff
@@ -27,7 +27,7 @@ def each(&block)
     def changes
       @changes ||= begin
         diff, _, result = Dir.chdir(@repo_root) do
-          Open3.capture3("git", "diff", "--name-only", @base_branch)
+          Open3.capture3("git", "diff", "--name-only", "--merge-base", @base_branch)
```

Comment:
I think we need some test coverage to demonstrate the bug.

Reply:
Fixture modified to demonstrate the bug